BreadPlayer.Views.UWP/Shell.xaml.cs

```diff
@@ -44,20 +44,18 @@ public Shell()
             this.InitializeComponent();
             SurfaceLoader.Initialize(ElementCompositionPreview.GetElementVisual(this).Compositor);
             CoreApplication.GetCurrentView().TitleBar.ExtendViewIntoTitleBar = true;
-            CoreWindowLogic logic = new CoreWindowLogic();
```

Comment:
This is important here as it initializes the key part of the CoreWindowLogic. Removing this will break things.